templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
```

Comment:
The use of `{% if user.is_authenticated %}` is correct for conditionally displaying content based on the user's authentication status. Ensure that the `user` context variable is available in the templates where this include is used.